sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/KeyVaultBackupAsyncClient.java

```diff
@@ -400,6 +402,102 @@ private static Mono<PollResponse<KeyVaultBackupOperation>> processBackupOperatio
+     * <pre>
+     * String blobStorageUrl = &quot;https:&#47;&#47;myaccount.blob.core.windows.net&#47;myContainer&quot;;
+     * String sasToken = &quot;sv=2020-02-10&amp;ss=b&amp;srt=o&amp;sp=rwdlactfx&amp;se=2021-06-17T07:13:07Z&amp;st=2021-06-16T23:13:07Z&quot;
+     *     + &quot;&amp;spr=https&amp;sig=n5V6fnlkViEF9b7ij%2FttTHNwO2BdFIHKHppRxGAyJdc%3D&quot;;
+     *
+     * client.beginPreBackup&#40;blobStorageUrl, sasToken&#41;
+     *     .setPollInterval&#40;Duration.ofSeconds&#40;1&#41;&#41; &#47;&#47; You can set a custom polling interval.
+     *     .doOnError&#40;e -&gt; System.out.printf&#40;&quot;Pre-backup check failed with error: %s.%n&quot;, e.getMessage&#40;&#41;&#41;&#41;
+     *     .doOnNext&#40;pollResponse -&gt;
+     *         System.out.printf&#40;&quot;The current status of the operation is: %s.%n&quot;, pollResponse.getStatus&#40;&#41;&#41;&#41;
+     *     .filter&#40;pollResponse -&gt; pollResponse.getStatus&#40;&#41; == LongRunningOperationStatus.SUCCESSFULLY_COMPLETED&#41;
+     *     .flatMap&#40;AsyncPollResponse::getFinalResult&#41;
+     *     .subscribe&#40;unused -&gt; System.out.printf&#40;&quot;Pre-backup check completed successfully.%n&quot;&#41;&#41;;
+     * </pre>
+     * <!-- end com.azure.security.keyvault.administration.KeyVaultBackupAsyncClient.beginPreBackup#String-String -->
+     *
+     * @param blobStorageUrl The URL for the Blob Storage resource where the backup will be located.
+     * @param sasToken Optional Shared Access Signature (SAS) token to authorize access to the blob. If {@code null},
+     * Managed Identity will be used to authenticate instead.
+     *
+     * @return A {@link PollerFlux} polling on the {@link KeyVaultBackupOperation pre-backup operation} status.
+     *
+     * @throws KeyVaultAdministrationException If the given {@code blobStorageUrl} or {@code sasToken} are invalid.
+     * @throws NullPointerException If the {@code blobStorageUrl} is {@code null}.
+     */
+    @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
+    public PollerFlux<KeyVaultBackupOperation, String> beginPreBackup(String blobStorageUrl, String sasToken) {
+        if (blobStorageUrl == null) {
+            throw LOGGER.logExceptionAsError(
+                new NullPointerException(
+                    String.format(KeyVaultErrorCodeStrings.PARAMETER_REQUIRED, "'blobStorageUrl'")));
+        }
+
+        return new PollerFlux<>(getDefaultPollingInterval(),
+            preBackupActivationOperation(blobStorageUrl, sasToken),
+            backupPollOperation(),
+            (pollingContext, firstResponse) ->
+                Mono.error(LOGGER.logExceptionAsError(new RuntimeException("Cancellation is not supported"))),
+            backupFetchOperation());
+    }
+
+    /**
+     * Initiates a full backup of the Key Vault. This operation checks whether a full backup operation can be performed
```

Comment:
Should this match the `beginPreBackup` docstring? I do see comments in the code regarding backups, so does this actually attempt the backup?
```suggestion
     * Initiates a pre-backup of the Key Vault. This operation checks whether a full backup operation can be performed
```

Reply:
Good catch!